paper.md

```diff
@@ -138,25 +142,20 @@ Several hosted CI services like Travis, Circle, and Jenkins make continuous inte
 YAML [@ben2009yaml] is a human-readable data-serialization language. 
 It is commonly used in writing configuration files and in applications where data is stored or transmitted. 
 Due to its simplicity and wide adoption [@yaml_wide_adoption], we chose YAML for defining popper workflows and for specifying the configuration for the execution engine. 
-An example of a popper workflow is shown below.
+An example popper workflow is shown in @Lst:wf-example.
```

Comment:
For the "expand listing caption" todo item, we can explain what each step is doing in more detail.